LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2016-2023, EPFL
```

Comment:
the repo was [initialized in 2018](https://github.com/scalacenter/sbt-scalafix/commit/aee6a4f1f77fca8f4349f715178ca86835d7c76f) but first release of sbt-scalafix is from 2016, [from the scalafix repo](https://github.com/scalacenter/scalafix/releases/tag/v0.1.0).